ExtraDry/ExtraDry.Server/ExtensionMethods/QueryableExtensions.cs

```diff
@@ -9,7 +11,13 @@ namespace ExtraDry.Server;
 /// and paging API result sets.
 /// </summary>
 public static class QueryableExtensions {
-    
+
+    static QueryableExtensions()
+    {
+        var options = StaticServiceProvider.Provider?.GetService<IOptions<ExtraDryOptions>>()?.Value ?? new ExtraDryOptions();
```

Comment:
This feels like it could hide a number of issues for the consumer and not report back any warnings that could help a developer.

Reply:
Removed to couple more tightly rather than the static Service Provider. Neither are great, but doesn't hide the issue